cmd/kwil-admin/nodecfg/toml.go

```diff
@@ -166,20 +168,17 @@ pg_db_name = "{{ .AppConfig.DBName }}"
 # any TCP listen address.
 admin_notls = {{ .AppConfig.NoTLS }}
```

Comment:
Isn't this totally unnecessary now since tls only gets configured for admin?

At what point would anyone set this to true?

Reply:
So this can change, but right now TLS automatically gets enabled if the listen address is not either unix or a loopback tcp.  The cert/key being empty in the config does not disable it (as it does with the user service) as the defaults to use in the aforementioned case are just rpc.cert/rpc.key in the root directory.
Thus, @charithabandi uses `admin_notls` to bypass this so test containers' admin service can be accessed via non-loopback TCP.

I'll take another a look but I think we want to enable TLS if it's listening on an interface that looks like it's going to be accessed from a remote machine. Two types of authentication can be used: password with `admin_pass`, which dangerous if the transport is not encrypted, or mTLS.

Will look again.